pkgs/games/ldmud/default.nix

```
@@ -0,0 +1,107 @@
+{ lib, fetchFromGitHub, stdenv, autoreconfHook, pkg-config, bison, libiconv
+, pcreSupport ? true, pcre, gcryptSupport ? true, libgcrypt, erqSupport ? "erq"
+, compatModeSupport ? false, filenameSpacesSupport ? false, ipv6Support ? false
+, mccpSupport ? false, zlib, mysqlSupport ? false, libmysqlclient
+, postgresSupport ? false, postgresql, sqliteSupport ? false, sqlite
+, jsonSupport ? false, json_c, xmlSupport ? false, libxml2
+, deprecatedSupport ? true, tlsSupport ? false, openssl, pythonSupport ? false
+, python39, pythonSupportVersion ? python39 }:
```

```suggestion
, python39 }:
```
We can't use python 3.10?

---

There was an upstream regression in 3.10 that caused me some trouble when I first started this derivation, but that was months ago and I reported the bug upstream and it was fixed. I will swap this out for python310 and do some testing to make sure it's working as intended.

In the spirit of your other feedback I'll drop the `pythonSupportVersion` and just use python310 implicitly.